salt/modules/status.py

```diff
@@ -426,6 +426,9 @@ def meminfo():
     .. versionchanged:: 2016.11.4
         Added support for AIX
 
+    .. versionchanged:: 2017.7.3
```

Comment:
Can you change this to `Oxygen` instead of `2017.7.3`?

Reply:
Fixed :)